src/orange/orangeinp/UnitProto.cc

```diff
@@ -436,7 +403,7 @@ void UnitProto::build(ProtoBuilder& input) const
         // parent-reference-frame bbox
         auto local_bbox = apply_transform(calc_inverse(iter->second.transform),
                                           result.volumes[vol_id.get()].bbox);
-        input.expand_bbox(iter->second.univ_id, bump_bbox(local_bbox));
+        pb.expand_bbox(iter->second.univ_id, bump_bbox(local_bbox));
```

Comment:
Do these bboxes even matter now? You might be able to delete all that related code.